core/src/main/java/tech/pegasys/eth2signer/core/multikey/metadata/YamlSigningMetadataFileProvider.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.core.multikey.metadata;
+
+import tech.pegasys.eth2signer.core.multikey.SignerType;
+
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
+
+public class YamlSigningMetadataFileProvider implements SigningMetadataFileProvider {
+  private static final Logger LOG = LogManager.getLogger();
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper(new YAMLFactory());
```

Comment:
nit: must this be static?

Reply:
No but it's better to be static as creating the object mapper is a little bit expensive. Not something we want create everytime ideally